src/redux/modules/groups/groups.js

```diff
@@ -50,15 +55,15 @@ export default function groupsReducer(state = initialState, action = {}) {
       };
 
     case LOAD_GROUPS_SUCCESS:
-      entities = action.result;
+      entities = mergeResponse(state.groups.entities, action.result);
```

Comment:
Why not just rewrite the store from scratch every time we fetch the `index` endpoint?

I understand that this maybe be useful while fetching the `show` endpoint, but for `index` I'm worried about data leaks and out of synch data. It seems an early optimization to me.